charts/kyverno/templates/validate.yaml

```diff
@@ -1,3 +1,9 @@
 {{- if hasKey .Values "mode" }}
   {{ fail "mode is not supported anymore, please remove it from your release and use replicaCount instead." }}
 {{- end }}
+
+{{- if .Values.replicaCount }}
+  {{- if eq .Values.replicaCount 2 }}
+    {{ fail "Kyverno does not support running with 2 replicas." }}
```

Comment:
How about we add as a second sentence:  "For a highly-available deployment, select 3 replicas or for standalone select 1 replica." Does this work?

Reply:
Good idea. I will add that later today.

Reply:
Done